src/styles/utils/variables.scss

```
@@ -0,0 +1,10 @@
+$cell-size: 100px;
+$gap: 1px;
+$days-in-a-week: 7;
+$padding: 10px;
+$max-wight: $cell-size * $days-in-a-week + 2 
+  *$padding + $gap * ($days-in-a-week - 6);
+$days-week:
+  "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4,
+  "sat" 5, "sun" 6;

```

this is not wrong, but this is how it is customary to write values or function names in imports, if they do not fit on one line, then move each value to a new line. This will help in editing or copying in the future

```suggestion
$days-week:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```

---

Ok.